sdk/metricsadvisor/Azure.AI.MetricsAdvisor/samples/README.md

```diff
@@ -5,7 +5,6 @@ languages:
 products:
 - azure
 - azure-cognitive-services
-- azure-metrics-advisor
```

Comment:
I'm not following. Why is removing this needed?

Reply:
Let's go with `azure-cognitive-services` for now. I can update it again once we have the `azure-metrics-advisor` ID.